### DISCUSSION TITLE
Does the Xournal++ community have its own discord?

### DISCUSSION THREAD

**Discussion by K4LCIFER** [xournalpp/xournalpp#2801] [Q&A]
I've observed that other open source communities have their own discord servers for their projects, but I haven't seen existence of one for Xournal++.

If it doesn't exist, I would say that it would be handy to have to help people solve the small Q&A Questions about Xournal++ instead of opening up a full issue for it on GitHub.

**Comment by Technius** [comment]
We don't, but this is exactly what GitHub discussions are for ;)

You can also our Gitter or Matrix if you need chat. [See our (incomplete) website for some additional community resources.](https://xournalpp.github.io/community/help/)
